addons/social_share/wizards/snshare_url_multi.py

```diff
@@ -0,0 +1,50 @@
+from markupsafe import Markup
+
+from odoo import fields, models
+
+
+class ShareWizard(models.TransientModel):
+    _name = 'snshare.url.share.multi'
+    _description = 'Email Social Share Campaign Wizard'
+
+
+    share_campaign_id = fields.Many2one('snshare.campaign', string='Social Share Campaign', ondelete='cascade', required=True)
+    domain = fields.Char(default="[]")
+    subject = fields.Char()
+    message = fields.Html()
+    model = fields.Char(related='share_campaign_id.model_id.model')
+
+    def action_send(self):
+        composer = self.env['mail.compose.message'].create({
+            'composition_mode': 'mass_mail',
+            'force_default_recipients': True,
+            'model': self.model,
+            'res_domain': self.domain or '[]',
+            'template_id': self.share_campaign_id.mail_template_id.id,
+        })
+        # TODO test this
+        composer.write({
+            'body': composer.body.replace(Markup('<div id="message"></div>'), Markup(f'<div id="message">{self.message}</div>')),
```

Comment:
Ok, So now I get the format of default template on campaigns :) You create it but override when using the share wizard